### DISCUSSION TITLE
Is there a quick way to adjust all sub lines from one point to end of file

### DISCUSSION THREAD

**Discussion by Ronstang** [SubtitleEdit/subtitleedit#6309] [Q&A]
I am running into a lot of subtitles that are perfectly synched until one point then that particular line is off, usually late.  I tried point sync but that adjusts all lines in the entire file.  I just want a minor half second or so adjustment at a single point that also adjusts everything after that at the same time.  I tried selecting everything from that point to the end and selecting "adjust all times" but I don't understand how to get it to move the exact amount I want so it becomes trial and error and time consuming.

Any clarification would be greatly appreciated.  I love this program and have found it very useful lately.  I never knew how much it could do.....but it appears I need to learn a little more.

**Comment by darnn** [comment]
Yes. Under Adjust (in the lower left corner menu), "Set start and offset the rest". You can also configure a keyboard shortcut in Settings->Shortcuts.

**Comment by LeonCheung** [comment]
By default, the hidden trick is to click on waveform while holding Ctrl + Shift.

**Reply by Ronstang** [reply]
Thanks, that worked great.....learning this program fast.  Only problem is now knowing what I can do with all these subs its gonna take me years to catch up.
